### ISSUE TITLE
Install chromium extensions to notion-repackaged app

### ISSUE THREAD

**Issue by Mikestriken** [notion-enhancer/desktop#915]
**What makes the feature necessary?**
Some features were rejected because 'a chromium extension' is a better solution to this problem. People using the notion-repackaged app, don't have access to chromium extensions

**How would the feature work?**
1. User specifies extension folder where chromium extensions are located
2. Extensions can be enabled / disabled / named in notion enhanced config (maybe in their own tab next to core, extensions, themes etc)

**Comment by dragonwocky** [comment]
Hi there, thanks for all the suggestions. Unfortunately this particular one is out of scope. Notion desktop is built with Electron, which supports a subset of the Extensions API for development purposes, but does not have or aim to have full compatibility equivalent to a browser (you can read more about that [here](https://www.electronjs.org/docs/latest/api/extensions)).

The notion-enhancer itself can't run the same code 1:1 for the desktop version and browser extension. There's an additional layer required to wrap around running mods to adapt to the environment they're running in - which a browser extension not built specifically to run in an Electron app wouldn't have.

Can you link me to which specific features you're referring to? I don't remember rejecting any that would have been relevant for notion-repackaged purely because a Chrome extension was a better solution.

**Comment by Mikestriken** [comment]
> Can you link me to which specific features you're referring to? I don't remember rejecting any that would have been relevant for notion-repackaged purely because a Chrome extension was a better solution.

Namely [this one](https://github.com/notion-enhancer/repo/issues/122).

**Comment by dragonwocky** [comment]
I think you misunderstood my response there. The feature was rejected because it's out of scope for the notion-enhancer. Integrating that deeply with Notion's text editing involves fighting for control of the page at the highest level, and it's very easy to end up with corrupted pages and data loss because of it - let alone working out how to actually generate the suggestions without paying a subscription fee. This is a feature that I'm leaving up to Notion to work out.

I suggested LanguageTool, Grammarly and Notion AI as alternatives the person asking could try out. Not as the reason for the feature being rejected. Closed issues are generally closed for good reason, please don't repeat requests for rejected features.

**Comment by Mikestriken** [comment]
> I think you misunderstood my response there. The feature was rejected because it's out of scope for the notion-enhancer. Integrating that deeply with Notion's text editing involves fighting for control of the page at the highest level, and it's very easy to end up with corrupted pages and data loss because of it - let alone working out how to actually generate the suggestions without paying a subscription fee. This is a feature that I'm leaving up to Notion to work out.
> 
> I suggested LanguageTool, Grammarly and Notion AI as alternatives the person asking could try out. Not as the reason for the feature being rejected. Closed issues are generally closed for good reason, please don't repeat requests for rejected features.

I think you misunderstood my original request as well. I understood why the issue was rejected, but I thought your alternative was a good idea / workaround. The issue was and is however the fact that that I am using notion-repackaged and not the browser extension.

To be clear, I don't even desire the feature mentioned in that particular issue, rather I stumbled across it while trying to resolve my bug report [here](https://github.com/notion-enhancer/notion-enhancer/issues/920).

But when I saw your suggestion in that issue I added the issue here to my idea list of features.

Again, this issue is entirely separate from that old issue that was closed. The only relation between that issue and this one is that I got the idea for this new and completely separate feature from that old issue. The only reason I linked where I saw you "reject" an issue and cited a workaround with extensions was to sate your curiosity and answer the question you asked. Regardless, both issues have been closed because they are not feasible to be implemented.

------------------------
On a side note, I take it this means it would also be impossible to implement a move cursor to next word sub part.

Basically in most text editors (notion included) Ctrl + Left / Right arrow moves the text editor cursor to the next / previous word. However in most editors (notion included) it doesn't account for camel cased words.

Basically [this old stack overflow request for VSCode](https://stackoverflow.com/questions/314095/make-visual-studio-understand-camelcase-when-hitting-ctrl-and-cursor-keys) is also not possible.
